### PR TITLE
PMP: Fix in normalize()

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes

If the normal vector is small enough, it might happen that the vector is considered not null, but the approximate_sqrt of its norm is. Then the returned normalized vector is Nan. To avoid that, i added a check that the norm is not null.

## Release Management

* Affected package(s):PMP

### CODE REVIEW

File: Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/compute_normal.h [hunk 1]
```diff
@@ -266,7 +266,10 @@ compute_vertex_normal(typename boost::graph_traits<PolygonMesh>::vertex_descript
     he = opposite(next(he, pmesh), pmesh);
   } while (he != end);
 
-  if ( ! traits.equal_3_object()(normal, CGAL::NULL_VECTOR))
+  if(normal.x() != normal.x()
```
Comment (sloriot) [inline Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/compute_normal.h]: You are normalizing only if one coord is a NAN?

Comment (maxGimeno) [reply Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/compute_normal.h]: Arf I thought I removed that.

Comment (maxGimeno) [reply Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/compute_normal.h]: It is acually absolutely not the right commit

File: Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/compute_normal.h [hunk 2]
```diff
@@ -49,7 +49,9 @@ namespace internal {
   {
     typename GT::FT norm = CGAL::approximate_sqrt(
         traits.compute_squared_length_3_object()(v));
-    v = traits.construct_divided_vector_3_object()(v, norm);
+    //If the vector is small enough, approx_sqrt might return 0, and then we get nan values. 
+    //To avoid that, we check the resulted norm. If it is 0, we don't normalize.
+    v = traits.construct_divided_vector_3_object()(v, norm == 0 ? 1 : norm);
```
Comment (janetournois) [inline Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/compute_normal.h]: why don't you just use a `if(norm != 0)` ?
It avoids doin another construction

Comment (maxGimeno) [reply Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/compute_normal.h]: Indeed.

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-4.14-Ic-157.shtml

**Pull request merged**